L1Trigger/L1TGlobal/python/l1tGlobalSummary_cfi.py

```diff
@@ -0,0 +1,14 @@
+#
+import FWCore.ParameterSet.Config as cms
+
+l1tGlobalSummary = cms.EDAnalyzer("L1TGlobalSummary",
+		                  AlgInputTag = cms.InputTag("gtStage2Digis"),
+                                  ExtInputTag = cms.InputTag("gtStage2Digis"),
+		                  ## ExtInputTag = cms.InputTag("gtExtFakeProd"),
+		                  MinBx          = cms.int32(-2),
+		                  MaxBx          = cms.int32(2),
+		                  DumpRecord   = cms.bool(False), # print raw uGT record
+                                  DumpTrigResults= cms.bool(False),
+                                  DumpTrigSummary= cms.bool(True),
+                                  ReadPrescalesFromFile= cms.bool(False)
+)
```

Comment:
L1TGlobalSummary has a fillDescriptions() method, so should NOT have such an explicit cfi file.
This should be removed.